kms/tpmkms/tpmkms.go

```diff
@@ -73,21 +74,34 @@ type TPMKMS struct {
 }
 
 type algorithmAttributes struct {
-	Type  string
-	Curve int
+	Type     string
+	Curve    int
+	Requires []tpm2.Algorithm
```

Comment:
See remark about having to import `tpm2`

Reply:
657022a